.drone.star

```diff
@@ -39,6 +39,7 @@ dir = {
     "ocisRevaDataRoot": "/srv/app/tmp/ocis/owncloud/data/",
     "federatedOcisConfig": "/var/www/owncloud/web/tests/drone/config-ocis-federated.json",
     "ocmProviders": "/var/www/owncloud/web/tests/drone/providers.json",
+    "chromiumZip": "/var/www/owncloud/web/playwright-chromium.tar.gz",
```

Comment:
```suggestion
    "playwrightBrowsersArchive": "/var/www/owncloud/web/playwright-browsers.tar.gz",
```

Reply:
whats the problem with chromium?

Reply:
the archive is of folder where browsers are installed by playwright not only chromium
Should we create another path entry for other browsers? like `playwright-firefox.tar.gz`